### PR TITLE
removing old information

### PR DESCRIPTION
**Pull request by BriannaGreene**
Should we remove old instructions to reduce confusion?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,45 +43,9 @@ for more details about the pipeline.
...
-#### Shipping
-To deploy all shipping services, you can use the following with the desired cluster name:
-```bash
-jake deploy:shipping['stag','uat']
-```
-
 ### ZSH Users Special Syntax
 According to the [official documentation](http://jakejs.com/docs), ZSH users may need to do one of two things to run `jake` commands:
-- Escape brackets or wrap them in single quotes and omit inner quotes: `jake 'deploy:app[stag,uat,console]'`
+- Escape brackets or wrap them in single quotes and omit inner quotes: `jake 'pipeline:deploy['stag','one','console','my-branch']'`
```
Comment (tom-con) [inline README.md]: I think this needs to have the quotes removed... Not sure if zsh still has this problem, but I remember we had to remove inner quotes and add the outer quotes.

Comment (tom-con) [reply README.md]: Do you know if this is the case?

Comment (BriannaGreene) [reply README.md]: I'm honestly not sure about this, I don't have this issue, even though I do use zsh.

**Comment by tom-con** [comment]
> The old way still works, so I think we should leave it in the docs until it is truly obsolete. I do like the update for the ZSH syntax, though.

oh true we do still support non-pipeline deployments

File: README.md (hunk 1 above)
Comment (tom-con) [reply README.md]: ah ok, well if it's not a problem anymore then sounds like it can go away

**Comment by BriannaGreene** [comment]
ok, closing this.

**Pull request closed without merging**